### PR TITLE
Add dem env installer script

### PR DESCRIPTION
**Pull request by magyta93**
## Type Of Change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Any modification in the test cases
- [x] Any modification in the documentation

### Checklist:

- [x] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
- [x] I have checked to ensure there aren't other open Pull Requests for the same update/change.
- [x] All the test cases pass and new modifications in the production code are 100% covered.
- [x] I have made corresponding changes to the documentation.

## Related Issue
Closing: 
[DEM-141](https://axem.atlassian.net/browse/DEM-141) 
#129

## Description
I created an installer script for install axem-dem pip module and docker. Before start the installation the script check the python version of the system.

## How Has This Been Tested?
It was tested on a freshly installed Debian 12 with Python 3.11

[DEM-141]: https://axem.atlassian.net/browse/DEM-141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,15 @@ DEM is available in the [PyPI repository](https://pypi.org/project/axem-dem/). I
 
 :information_source: The package name is axem-dem, but the command is `dem`.
 
+
+You can also install it with init-dem.sh from our repository. The installer script checks are you using the corresponding python version and install docker.io and axem-dem pip modul. Download it with the the following command:
```
Comment (janosmurai) [inline README.md:95]: Rephrase to "The installer script checks if the corresponding python version is available and installs the docker and axem-dem pip modul.

(New line)
Download it with the following command:"

File: docs/installation.md [hunk 2]
```diff
@@ -21,6 +21,15 @@ And that's it! Now you should be able to use the `dem` command.
 
     The package name is axem-dem, but the command is `dem`.
 
+
+You can also install it with init-dem.sh from our repository. The installer script checks are you using the corresponding python version and install docker.io and axem-dem pip modul. Download it with the the following command:
```
Comment (janosmurai) [inline docs/installation.md:25]: Rephrase to "The installer script checks if the corresponding python version is available and installs the docker and axem-dem pip modul.

(New line)
Download it with the following command:"

**Review by janosmurai — CHANGES_REQUESTED** [review]
Please rename the init-dem.sh file to install-dem.sh and update all references in the documentation accordingly.

File: README.md (hunk 1 above)
Comment (magyta93) [reply README.md]: fixed!

File: docs/installation.md (hunk 2 above)
Comment (magyta93) [reply docs/installation.md]: fixed!

**Review by janosmurai — APPROVED** [review]

**Pull request merged**